Framework/API/src/NotebookWriter.cpp

```diff
@@ -180,11 +172,9 @@ Json::Value NotebookWriter::buildNotebook() {
  * the whole notebook
  */
 std::string NotebookWriter::writeNotebook() {
-
   const Json::Value root = buildNotebook();
 
-  Json::StyledWriter writer;
-  std::string output_string = writer.write(root);
+  std::string output_string = Mantid::JsonHelpers::jsonToString(root, " ");
```

Comment:
Do we need the temporary?
```suggestion
return Mantid::JsonHelpers::jsonToString(root, " ");
```